workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+            </p>
+
+            <p>
+              <label for="sepia">Sepia</label>
+              <input id="sepia" name="sepia" type="range" min="0" max="" value="0" class="slider">
+            </p>
+            
+            <button type="reset" class="btn btn-primary" >Reset</button>
+        </form>
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+</div>
+```
+</details>
+
+![Section 3 Result](https://cloud-jwbtp72pu.vercel.app/0screen_shot_2020-11-11_at_12.33.02_pm.png)
+
+# CSS
+The Boostrap stylesheet we added at the beginning of this workshop gives us most of the styling we need. But let's do a little bit of our own customizing.
+
+Open the `style.css` file on the sidebar on the left. Then, add:
+
+```css
+img {
+  width: 400px;
+}
+
+#section_three {
+  border: 2px solid black; 
+}
+```
+Let's break this down:
+- First, we set `400px` as the default width for every image we edit because some images can be too big and otherwise might not fit on the website.
+- Then, we add a border to the section 3 to make it look a little bit better.
+
+Here's the result:
+![CSS Result](https://cloud-i0vj1gzw6.vercel.app/0image.png)
+
+Repl.it should have automatically linked the `style.css` file for you, but if you don't see your changes reflect when you click the green "Run" button at the top, try adding this line to the `<head>` in your `index.html` file:
+
+```html
+<link rel="stylesheet" href="style.css">
+```
+
+# JavaScript
+This is my favorite part of the workshop, because this part makes the website actually work.
+
+Navigate to the `script.js` file in your sidebar. Then, create these 3 variables:
+
+```javascript
+let myform = document.getElementById('imgurl');
+let targetimage = document.getElementById('targetimage');
+let inputrange = document.querySelectorAll('.slider');
+```
+- The first and second variable will hold html elements, we get those elements using the method ```document.getElementById('')```. So those variables will hold the form in section 1 and the image in section 2 respectively.
+- In the third variable, you'll find `document.querySelectorAll` returns a NodeList, a special type of list of elements, that will contain the amount of elements with  `class=slider` in the document. So the 3rd variable contains the value of all the range inputs in the section 3.
+
+Next, we want to trigger some JavaScript code to run when the form is submitted. Under the three variables, add the following code:
+
+```js
+myform.addEventListener('submit', function(e) {
+  let urlimage = document.getElementById('onlineurl');
+  let urlimageval = urlimage.value;
+  if(urlimageval.length){
+    targetimage.setAttribute('src',urlimageval);
+      urlimage.value = '';
+  }
+    
+  e.preventDefault();
+  console.log('Image displayed');
+});
+```
+Let's break this down:
+We are using the method ```addEventListener()``` to the first variable that we created, when the button in the form is submitted we call a function that:
```

Comment:
Should add an empty line between these two lines. Right now it renders weirdly in the HTML